src/components/Team/Team.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import Img from 'gatsby-image'
+
+import { FaInstagram, FaGithub, FaLinkedin, FaTwitter } from 'react-icons/fa'
+import { Row, Col, Card, CardBody, CardImg, CardTitle, CardSubtitle, CardText } from 'reactstrap'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Socials } from '../../components'
+
+export const Team = props => {
+  const listOfFiles = useStaticQuery(imageQuery).allFile.edges
+  const allFiles = Object.assign(
+    {},
+    ...listOfFiles.map(item => ({ [item.node.name]: item.node.childImageSharp.fluid }))
+  )
+  return (
+    <Row className="row-eq-height">
+      {props.team
+        .sort((a, b) => a.listOrder - b.listOrder)
+        .map(member => (
+          <Col md="12" lg="4" key={member.id} className="mb-3">
+            <Card className="shadow-sm bg-white h-100">
+              <CardBody>
+                <Img fluid={allFiles[member.name]} style={{ margin: '0.5em' }} />
```

Comment:
remove margin or just use `{ margin: '0.5em 0'}` for top and bottom margin

Reply:
Done!